app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Contestants and Projects</h1>
+<% @contestants.each do |contestant| %>
+  <h3><%= contestant.name %></h3>
+        <% contestant.projects.each do |project| %>
```

Comment:
Great work drilling into the contestant object to get the projects associated with it!